src/Fonts.php

```diff
@@ -9,11 +9,13 @@
 class Fonts implements Htmlable
 {
     public function __construct(
-        protected string  $googleFontsUrl,
-        protected ?string $localizedUrl = null,
-        protected ?string $localizedCss = null,
-        protected ?string $nonce = null,
-        protected bool    $preferInline = false,
+        protected string    $googleFontsUrl,
+        protected ?string   $localizedUrl = null,
+        protected ?string   $localizedCss = null,
+        protected ?string   $nonce = null,
+        protected bool      $preferInline = false,
```

Comment:
Can you avoid the whitespace changes?

Reply:
Absolutely, I've made sure to avoid any whitespace changes in the commit.